### DISCUSSION TITLE
最后一次在绳网给大伙发你们想看的东西了

### DISCUSSION THREAD

**Discussion by fzdqz** [share121/inter-knot#5560] [公告]
大家且看且珍惜吧，再会了
选项1有熊很大的本

**Comment by RWHwaxili** [comment]
你这网站不对劲

**Reply by fzdqz** [reply]
哪里不对劲了😡你家本子不是用网盘分享的？

**Reply by RWHwaxili** [reply]
要就发123盘或者蓝奏云，这个我不太认可

**Reply by fzdqz** [reply]
> 要就发123盘或者蓝奏云，这个我不太认可

这就是123云盘单个文件分享的状态啊<img src="https://img-static.mihoyo.com/communityweb/upload/8b198c0a467614f263e08593bd4b000d.png" alt="安比_无语" width="75" >

**Reply by RWHwaxili** [reply]
不是123pan.com开头吗

**Reply by fzdqz** [reply]
> 不是123pan.com开头吗

不知道啥情况，总之打开就是我分享的那张图

**Comment by xintie2023** [comment]
看成熊大的本了。

**Comment by StellaBigHero** [comment]
楼主真是好人一生平安，工地打灰必被熊希人贯穿

**Comment by cpu644** [comment]
熊很大

**Comment by rtcrtc123** [comment]
？
